src/libraries/pkg/test/frameworkSettings/netcoreapp1.0/settings.targets

```diff
@@ -1,32 +0,0 @@
-<Project>
-  <ItemGroup Condition="'$(RuntimeIdentifier)' != ''">
```

Comment:
I'm not sure I follow Why these can be removed.

Reply:
Because we no longer test with `RuntimeIdentifier` at all.